tox.ini

```diff
@@ -45,14 +45,15 @@ deps =
 commands =
     pre-commit run --all-files --show-diff-on-failure
 
-[testenv:types]
+[testenv:type]
 description = run type check on code base
 setenv =
     {tty:MYPY_FORCE_COLOR = 1}
 deps =
     mypy==0.910
 commands =
     mypy --strict src
```

Comment:
Isn't running both at once a bit faster?

```suggestion
    mypy --strict src tests
```

Reply:
Having it separate means one can first focus on getting the src right and only catter about the tests later. The performance overhead is negligable.